pkg/sippyserver/server.go

```diff
@@ -1278,7 +1278,7 @@ func (s *Server) jsonTriages(w http.ResponseWriter, req *http.Request) {
 		if triageID > 0 {
 			existingTriage, err := componentreadiness.GetTriage(s.db, triageID)
 			if err != nil {
-				failureResponse(w, http.StatusInternalServerError, err.Error())
+				failureResponse(w, http.StatusNotFound, err.Error())
```

Comment:
Wondering if you have seen / it is worth explicitly checking explicitly to see if the error is record not found:
`if !errors.Is(err, gorm.ErrRecordNotFound)` 

Could potentially be applied to https://github.com/openshift/sippy/blob/main/pkg/api/componentreadiness/triage.go#L15 and skip the error logging if it is reasonable for it not to be present.  If we don't expect that case then leaving the error logging is fine.

Reply:
This is a good idea. Normal navigation shouldn't result in `404's`, but if someone deletes the triage record, and then refreshes the details page, it will result in one. They could also get to that state by navigating to a triage details page manually (with a non-existant id) or by using an old link to one that has since been deleted. None of those are error states.